### DISCUSSION TITLE
Dark high contrast theme.

### DISCUSSION THREAD

**Discussion by HubKing** [dbeaver/dbeaver#20166] [Ideas]
There is a "high contrast" theme in the selection, but that does not seem to work when the OS is using a dark theme (it shows black text on a dark background).  The only working thing is "Dark", but to me, the background is too bright and lacks contrast. Look at the screenshot below. I can barely read the "Enter an SQL expression..." message.

Technically, I could change all the colours manually in Appearance -> Colours and.., but there are too many items. It would be nice if it had a pre-defined high contrast dark theme, just like JetBrain's IDE's or VS Code.

![image](https://github.com/dbeaver/dbeaver/assets/10900489/6070383e-404c-4ae0-8057-97938e9a930f)

![image](https://github.com/dbeaver/dbeaver/assets/10900489/739d61ad-5653-49d6-a4a7-f91b69cba597)

**Comment by E1izabeth** [comment]
Thank you for the idea
